content/blog/2025/10/2025-10-24-twim.md

```diff
@@ -0,0 +1,322 @@
+>
+> And just last weekend, as we are wrapping up [The Matrix Conference](https://conference.matrix.org) in Strasbourg, we receive word that Matrix has been [selected in the second tranche of 2025 to receive $25000](https://floss.fund/blog/second-tranche-2025-anniversary/)!
+>
+> We congratulate all of the other awesome projects who were selected! 🎉  
+> A million thanks to FLOSS/fund for selecting us, as well as everyone else involved for making this happen. 🤩  
+> We are excited to see where the road for FLOSS/fund leads next year! 🚀  
+> If you are able and looking to fund FLOSS projects you are building on, find worth supporting, or even just to discover new ones, we absolutely encourage you to browse the awesome directory at <https://dir.floss.fund/>!
+>
+> Find out how else you can support The Matrix.org Foundation at <https://matrix.org/support/>. Does your organisation rely on Matrix? [Join us as a funding member](https://matrix.org/membership/).
+
+### DINUM sponsors Matrix
+
+[Amandine](https://matrix.to/#/@Amandine:matrix.org) says
+
+> 🇫🇷 Very excited to announce that [DINUM, the French Interministerial Digital Directorate, is joining the Matrix.org Foundation](https://www.numerique.gouv.fr/sinformer/espace-presse/dinum-soutien-fondation-matrixorg/) as a Silver member and becomes the first government to join the Foundation! 🚀
+>
+> Matrix is the backbone of [Tchap](https://tchap.numerique.gouv.fr/), the secure messaging app for the French government, and it is great to see a State stepping up to concretely support and collaborate with an open source project so critical to their functioning.
+>
+> 💡 We hope this will show the way to all the other public sector organisations using Matrix, especially after we kicked off a Matrix for Public Sector group at the Matrix Conference last week, hopefully soon approved by the Governing Board as an official [Working Group](https://matrix.org/foundation/working-groups/) 🤞!
+>
+> ➡️ check out [the announcement blog](https://matrix.org/blog/2025/10/dinum/)!
+
+### Updated Support and Membership Pages
+
+[HarHarLinks](https://matrix.to/#/@kim:sosnowkadub.de) reports
+
+> Just in time for The Matrix Conference, the Website & Content WG have revamped the /support and /membership pages of matrix.org for clearer structure, making it easier to find the methods to support the Foundation, and collecting all info about membership on one page instead of spreading it across multiple.
```

Comment:
```suggestion
> Just in time for The Matrix Conference, the [Website & Content WG](@/foundation/working-groups/index.md) have revamped the /support and /membership pages of matrix.org for clearer structure, making it easier to find the methods to support the Foundation, and collecting all info about membership on one page instead of spreading it across multiple.
```